storage/storage-service/src/lib.rs

```diff
@@ -75,33 +73,19 @@ pub struct StorageService {
 ///   https://github.com/facebook/rocksdb/issues/649
 struct LibraDBWrapper {
     db: Option<LibraDB>,
-    shutdown_sender: Mutex<mpsc::Sender<()>>,
 }
 
 impl LibraDBWrapper {
-    pub fn new<P: AsRef<Path>>(path: &P) -> (Self, mpsc::Receiver<()>) {
+    pub fn new<P: AsRef<Path>>(path: &P) -> Self {
         let db = LibraDB::new(path);
-        let (shutdown_sender, shutdown_receiver) = mpsc::channel();
-        (
-            Self {
-                db: Some(db),
-                shutdown_sender: Mutex::new(shutdown_sender),
-            },
-            shutdown_receiver,
-        )
+        Self { db: Some(db) }
     }
 }
 
 impl Drop for LibraDBWrapper {
```

Comment:
Can't we get rid of this now? The fields will get dropped when the container struct is dropped.